src/Gir.Tests/RecordTests.cs

```diff
@@ -28,6 +27,130 @@ public struct ByteArray
 
 		///<summary>the number of elements in the #GByteArray</summary>
 		uint len;
+
+		///<summary>
+		/// Adds the given bytes to the end of the #GByteArray.
+		/// The array will grow in size automatically if necessary.
+		///</summary>
+		///<returns>the #GByteArray</returns>
+		static void append PARAMS
```

Comment:
Is this right? Or is it just a place holder?

Reply:
Placeholder. not generating params/return value yet, I just wanted to make some building blocks first.